Denote availability of ASR for managed disk

andynava opened the pull request:
"Protect machines using managed disks between Azure regions using Azure Site Recovery"
https://azure.microsoft.com/en-us/blog/asr-managed-disks-between-azure-regions/

azurecla commented:
Hi __@jasonnewyork__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jahogg). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
Replaced by https://github.com/Azure/azure-content-pr/pull/17282

andynava committed:
Denote availability of ASR for managed disk

https://azure.microsoft.com/en-us/blog/asr-managed-disks-between-azure-regions/
